src/praisonai-bot/tests/unit/bots/test_tts_voice_reply.py

```
@@ -0,0 +1,176 @@
+    def test_over_max_chars_skips(self):
+        cfg = TtsConfig(enabled=True, max_chars=5)
+        assert synthesize_voice_reply("way too long", cfg) is None
+
+    def test_delegates_to_tts_tool(self, monkeypatch):
+        calls = {}
+
+        def fake_tts_tool(text, voice=None, model=None, output_format="ogg"):
+            calls.update(
+                text=text, voice=voice, model=model, output_format=output_format
+            )
+            return {"success": True, "audio_path": "/tmp/reply.ogg"}
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/reply.ogg"

(S108)